OpenEdXMobile/res/values/styles.xml

```diff
@@ -579,4 +580,10 @@
     <style name="edX.Widget.TextInputLayout.HintTextAppearance" parent="TextAppearance.AppCompat.Caption">
         <item name="android:textColor">@color/primaryBaseColor</item>
     </style>
+
+    <style name="sync.Calendar.Switch" parent="Theme.AppCompat.Light">
+        <item name="colorControlActivated">@color/neutralBlackT</item>
```

Comment:
as per design, it should be `primaryDarkColor(002121)`

Reply:
Done